app/models/user.rb

```diff
@@ -3,4 +3,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_many :bookings
+  has_many :tournaments
+  has_one_attached :photo
+  belongs_to :join_team_players
```

Comment:
i think it should be:

```suggestion
  has_many :join_team_players
```